neuralop/layers/fno_block.py

```diff
@@ -102,14 +102,13 @@ def __init__(
         preactivation=False,
         fno_skip="linear",
         channel_mlp_skip="soft-gating",
-        complex_data=False,
+        complex=False,
```

Comment:
complex_data was good - should we keep that?

Reply:
Reverted